code/wdb/datastripe.q

```diff
@@ -59,6 +59,32 @@ initdatastripe:{
 
 \d .ds
 
+symlink:{
+    /- function to create HDB sym file and symlink to this sym file at start up
+    /- create HDB sym file
+    symdir:`$raze string .wdb.hdbdir,"/sym";
+    if[not `sym in key .wdb.hdbdir;symdir set `symbol$()];
+    .lg.o[`hdbsym;"creating HDB symfile"];
+
+    /- create symlink
+    basedir:` sv .ds.td,.proc.procname;
+    if[not `sym in key basedir;createsymlink[basedir;.wdb.hdbdir;`sym]];
+    .lg.o[`symlink;"creating symlink"];
```

Comment:
same as above with the logging

Reply:
fixed the logging in both places now too